<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatePostView</class>
 <widget class="QWidget" name="CreatePostView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Post</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="titleInput">
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Raleway&quot;; padding: 10px; border-radius: 5px; border: 1px solid #ccc; color:white;</string>
     </property>
     <property name="placeholderText">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="fontSizeComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">    font: 500 11pt &quot;Raleway&quot;;
    border: 2px solid rgb(85, 170, 255);  /* Border color */
    background-color: transparent;  /* Remove background */
    color: white;  /* Text color */</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFontComboBox" name="fontComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 500 11pt &quot;Raleway&quot;; color:white;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolBar" name="formattingToolbar">
     <property name="styleSheet">
      <string notr="true">
       QToolBar {
           background: #f5f5f5;
           border-radius: 5px;
           padding: 5px;
           spacing: 5px;
       }
       QToolButton {
           background: transparent;
           border: 1px solid #ccc;
           border-radius: 3px;
           padding: 5px;
       }
       QToolButton:hover {
           background: #e0e0e0;
       }
       QToolButton:pressed {
           background: #d0d0d0;
       }
      </string>
     </property>
     <addaction name="boldAction"/>
     <addaction name="italicAction"/>
     <addaction name="underlineAction"/>
     <addaction name="strikeAction"/>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="bodyInput">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Raleway&quot;; padding: 10px; border-radius: 5px; border: 1px solid #ccc; color:white;</string>
     </property>
     <property name="placeholderText">
      <string>Text (optional)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="uploadImageButton">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Raleway&quot;; padding: 10px; border-radius: 5px; background-color: #0079D3; color: white;</string>
     </property>
     <property name="text">
      <string>Upload Image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="imagePreview">
     <property name="styleSheet">
      <string notr="true">border: 2px dashed #ccc; padding: 10px; border-radius: 5px;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="submitButton">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Raleway&quot;; padding: 10px; border-radius: 5px; background-color: #0079D3; color: white;</string>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="boldAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="iconText">
    <string>B</string>
   </property>
   <property name="toolTip">
    <string>Bold</string>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="italicAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="iconText">
    <string>I</string>
   </property>
   <property name="toolTip">
    <string>Italic</string>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="underlineAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
   <property name="iconText">
    <string>U</string>
   </property>
   <property name="toolTip">
    <string>Underline</string>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="strikeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Strike</string>
   </property>
   <property name="iconText">
    <string>S</string>
   </property>
   <property name="toolTip">
    <string>Strikethrough</string>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
